src/Numerics/DGmethods/balance_law.jl

```diff
@@ -86,6 +86,7 @@ function reverse_integral_set_auxiliary_state! end
 
 
 using ..Courant
+
```

Comment:
A bit orthogonal to this PR? I personally don't mind this but some might.

Reply:
This is a result of multiple commits getting squashed in the PR diff view. I can fix it though.